content/applications/inventory_and_mrp/inventory/routes/strategies/removal.rst

```diff
@@ -1,222 +1,377 @@
+FEFO (First Expired, First Out)
+-------------------------------
 
-FEFO (First Expiry, First Out)
-------------------------------
+The :abbr:`FEFO (First Expiry, First Out)` removal strategy differs from the :abbr:`FIFO (First In,
+First Out)` and :abbr:`LIFO (Last In, First Out)` strategies, because it targets products for
+removal based on **expiration dates** instead of their warehouse receipt dates. With this method,
+the products that are going to expire first are moved out first. This method is used for perishable
+products, such as medicine, food, and beauty products.
 
-With the *FEFO* strategy, the way products are picked is not based on the reception date. In this
-particular case, they are dispatched according to their expiration date.
+Lots are picked based on their **removal date** from earliest to latest. Removal dates indicate how
+many days *before* the expiration date the product needs to be removed from stock. The removal date
+is set on the product form. Lots without a removal date defined are picked after lots with removal
+dates.
 
-.. note::
-   To have more information about Expiration date, please have a look at
-   :doc:`the related doc <../../management/lots_serial_numbers/expiration_dates>`.
+.. warning::
+   If products are not removed from stock when they should be, lots that are past the expiration
+   date may still be picked for delivery orders!
 
-By activating *Expiration Dates*, it becomes possible to define different dates on the serial/lot
-numbers to be used in *FEFO*. These dates can be set by going to :menuselection:`Inventory -->
-Master Data --> Lots/Serial Numbers`.
+.. note::
+   For more information about expiration dates, reference the
+   :doc:`Expiration dates <../../management/lots_serial_numbers/expiration_dates>` document.
+
+First, go to :menuselection:`Inventory app --> Configuration --> Settings` and ensure
+:guilabel:`Expiration Dates` is enabled. Once the :guilabel:`Expiration Dates` setting is enabled,
+it's possible to define different expiration dates for individual serialized products, as well as
+for lot numbers containing many products.
+
+In this example, there are three lots of hand cream. The creams are from the All/Health & Beauty
+category, where :guilabel:`FEFO` is set as the removal strategy. In the :guilabel:`Inventory
+Valuation Report`, the three different receipts are listed with the amounts.
+
+Lot 0000001 contains twenty tubes of hand cream, expiring on Sept 30, lot 0000002 contains ten tubes
+of hand cream, expiring on November 30, and lot 0000003 contains ten tubes of hand cream, expiring
+on October 31.
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Lot/Serial No
+     - Product
+     - Expiration Date
+     - Amount In Stock
+   * - 0000001
+     - Hand Cream
+     - 09/30
+     - 20
+   * - 0000002
+     - Hand Cream
+     - 11/30
+     - 10
+   * - 0000003
+     - Hand Cream
+     - 10/31
+     - 10
```

Comment:
This table is great, but it feels out of place here since it's the only table in the doc, and all the other examples use images instead of tables. I would replace this with an image for consistency.